src/commands/time.js

```diff
@@ -19,6 +19,18 @@ const getCurrentTimes = () => {
 	return require(TIMER_FILE);
 };
 
+// logging function to be called when all metrics are available
```

Comment:
could you add a link to the api key settings https://app.datadoghq.com/account/settings#api